dev/workshop-syllabus.md

```diff
@@ -1,19 +1,19 @@
-# Tidy Transcriptomics [SOME OTHER TITLE?]
+# A Tidy Transcriptomics introduction to RNA-Seq
 
 Dr. Maria Doyle (EMAIL) and Dr. Stefano Mangiola (EMAIL)
 
 # Workshop Description
 
-This workshop will present how to perform analysis of RNA sequencing data following the tidiness paradigm. The tidy paradigm follows the representation of the data where each variable is a column and each observation is a row, and the manipulation of data using verbs with an easy-to-understand vocabulary. Most importantly the data structure can be forwarded across manipulation and analysis function keeping its properties (i.e., endomorphic). 
+This workshop will present how to perform analysis of RNA sequencing data following the tidy data paradigm. In the tidy data paradigm, data structures represent each variable as a column and each observation as a row, and data is manipulated using an easy-to-understand vocabulary. Most importantly, the data structure remains consistent across manipulation and analysis functions (i.e. it is endomorphic). 
 
-This can be achieved for RNA sequencing data with tidybulk, tidyHeatmap and tidyverse packages. The package tidybulk provides a modular framework and a tidy data structure for bulk transcriptional analyses. tidyHeatmap provides a tidy implementation of the function ComplexHeatmaps::Heatmap. These packages are part of the tidytranscriptomics suite that introduces semantics and properties of a tidy approach to RNA sequencing data.
+This can be achieved for RNA sequencing data with the tidybulk, tidyHeatmap and tidyverse packages. The package tidybulk provides a tidy data structure and a modular framework for bulk transcriptional analyses. tidyHeatmap provides a tidy implementation of ComplexHeatmap. These packages are part of the tidytranscriptomics suite that introduces a tidy approach to RNA sequencing data.
 
 The topics presented in this workshop will be
 
 - Data exploration
 - Data preprocessing and quality control
 - Data dimensionality reduction
-- Differential transcript abundance analyses 
+- Differential gene abundance analyses 
```

Comment:
I would prefer to say transcript, as gene abundance is technically incorrect. As gene is just a piece of information, gene abundance would be at most gene duplication on the genome.

Reply:
What about using differential gene expression instead? As wondering if differential transcript analysis might get confused with differential isoform analysis? e.g.  from [here](https://liorpachter.wordpress.com/2019/01/07/fast-and-accurate-gene-differential-expression-by-testing-transcript-compatibility-counts/) "Differential transcript expression (DTE) refers to change in one of the isoforms"